api-extractor/src/definitions/ApiPackage.ts

```diff
@@ -1,7 +1,7 @@
 /* tslint:disable:no-bitwise */
 
 import * as ts from 'typescript';
-import Analyzer from '../Analyzer';
+import Extractor from '../Extractor';
```

Comment:
This is a ton of diff churn.  In the future, renaming Analyzer should probably be a separate PR from new work